### ISSUE TITLE
Move away from bower

### ISSUE THREAD

**Issue by arthurhsu** [google/lovefield#167]
Lovefield needs to have a simpler package management. We should just use one package management system for our source code so that dependency management can be simplified.

One major reason we use bower before is that Polymer does not work well without bower. This is no longer the case.

**Comment by freshp86** [comment]
Can you be a bit more specific about what makes current situation complex? The overhead of supporting bower and NPM seems pretty light (maintaining a package.json, bower.json files). Bower is still pretty popular AFAIK so I would be hesitant to moving away from it unless there are concrete simplifications on the Lovefield side.

**Comment by arthurhsu** [comment]
I did not mean remove Lovefield's registration on bower. I mean remove Lovefield's internal usage of bower. So all dev process is streamlined as npm install and gulp, ruling out dependencies on bower.

There's not a feature in bower that Lovefield uses today not provided by npm.

**Comment by douglasg14b** [comment]
Agreed on this (1.5 years later), bower has died off and NPM has moved forward as the defacto package manager for front end dependencies... It just adds another barrier to entry for using the library.
